HUD/N_0x17ad8c9706bdd88a.md

```diff
@@ -8,6 +8,22 @@ ns: HUD
 void _0x17AD8C9706BDD88A(Any p0);
 ```
 
+Related to notification color flashing, setting p0 to 0 invalidates a `_SET_NOTIFICATION_FLASH_COLOR` call for the target notification.
+
+```c
```

Comment:
iirc, only 1 `c` codeblock can be used, and that's the one you see above this one. This code block should not be formatted explicitly using the `c` code style, just leave the highlighting out of this one.